GOVERNANCE-stage.md

```diff
@@ -0,0 +1,137 @@
+
+> In order to address the case of insufficient active binding voters to reach 2/3 majority, one can follow the process below to exclude a binding vote from the counting of this particular voting thread. 
+> 
+> 1. Wait until the minimum length of the voting passes.
+> 2. Publicly reach out via mentioned to the remaining binding voters in the voting pull request for at least 2 attempts with at least 7 days between two attempts.
+> 3. If the binding voter being contacted still failed to respond after all the attempts, the binding voter will be considered as inactive for the purpose of this particular voting.
+
+### Vetoes
+
+A valid, binding veto cannot be overruled. If a veto is cast, it must be accompanied by a valid reason explaining the reasons for the veto. The validity of a veto, if challenged, can be confirmed by anyone who has a binding vote. This does not necessarily signify agreement with the veto - merely that the veto is valid.
+
+If you disagree with a valid veto, you must lobby the person casting the veto to withdraw their veto. If a veto is not withdrawn, the action that has been vetoed must be reversed in a timely manner.
+
+### Actions
+
+Steering committee is permitted to take over team level decision making from the team maintainers, e.g., new maintainer or maintainer removal by consensus. However, it is a safety valve and the committee should avoid actually doing that.
+
+| Actions                  | Description                                                                                                                                                                                                                                         | Approval      | Binding Voters           | Minimum Length (days) |
+| :----------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ | :----------------------- | :-------------------- |
+| New Reviewer             | When a new reviewer is proposed for the team.                                                                                                                                                                                                       | Lazy Majority | Active maintainers       | 3                     |
+| New Committer            | When a new committer is proposed for the team.                                                                                                                                                                                                      | Consensus     | Active maintainers       | 3                     |
```

Comment:
New Committer and New Maintainer should have longer "Minimum Length"? I think it should be more than 6 days.

Reply:
and the New Committee Member too

Reply:
updated as 2f011a161e8f577a772012ba9937e08a03d83f12

Reply:
@BusyJay PTAL. I'm unsure whether it is a reasonable length for new committer.